frontend/pages/admin/IntegrationsPage/cards/CertificateAuthorities/CertificateAuthorities.tsx

```diff
@@ -83,26 +55,29 @@ const CertificateAuthorities = () => {
     setShowAddCertAuthorityModal(true);
   };
 
-  const onEditCertAuthority = (cert: ICertAuthority) => {
+  const onEditCertAuthority = (cert: ICertAuthorityListData) => {
     // TODO: use useCallback
-    const ca = getCertificateAuthority(
+    const certAuthority = getCertificateAuthority(
       cert.id,
       config?.integrations.ndes_scep_proxy,
       config?.integrations.digicert,
       config?.integrations.custom_scep_proxy
     );
-    console.log(ca);
+    setSelectedListItemId(cert.id);
+    setSelectedCertAuthority(certAuthority);
     setShowEditCertAuthorityModal(true);
   };
 
-  const onDeleteCertAuthority = (cert: ICertAuthority) => {
+  const onDeleteCertAuthority = (cert: ICertAuthorityListData) => {
     // TODO: use useCallback
```

Comment:
Is this todo still in progress?

Reply:
removing these comment as useCallback is not needed